pkg/apis/operator/v1alpha1/validation/validation.go

```diff
@@ -78,13 +78,25 @@ func validateOperationContext(operation string, garden *operatorv1alpha1.Garden,
 		if phase := helper.GetCARotationPhase(garden.Status.Credentials); len(phase) > 0 && phase != gardencorev1beta1.RotationCompleted {
 			allErrs = append(allErrs, field.Forbidden(fldPath, "cannot start rotation of all credentials if .status.credentials.rotation.certificateAuthorities.phase is not 'Completed'"))
 		}
+		if phase := helper.GetServiceAccountKeyRotationPhase(garden.Status.Credentials); len(phase) > 0 && phase != gardencorev1beta1.RotationCompleted {
```

Comment:
shoot validation uses `core.RotationCompleted`. Should we make it consistent?
https://github.com/gardener/gardener/blob/4c903c2ca8f03d6e1491f87f591a252f18021409/pkg/apis/core/validation/shoot.go#L2039

Reply:
This is not possible since the `Shoot` validation operates on the internal API of the `core.gardener.cloud` group while the `Garden` validation does not have an internal API.

Reply:
Ah, okay.